### PR TITLE
TK2.6: add services args to entity.casebook

### PR DESCRIPTION
**Pull request by frenchy64**
Epic: threatgrid/iroh#3986
Extracted from https://github.com/threatgrid/ctia/pull/924

<a name="qa">[§](#qa)</a> QA
============================

No QA is needed.

<a name="release-notes">[§](#release-notes)</a> Release Notes
=============================================================

```
intern: TK2.6: add services args to entity.casebook
```

<a name="squashed-commits">[§](#squashed-commits)</a> Squashed Commits
======================================================================

### CODE REVIEW

**Review by yogsototh — APPROVED** [review]
LGTM!

File: src/ctia/entity/casebook.clj [hunk 1]
```diff
@@ -130,7 +128,8 @@
     :delete-casebook
     :search-casebook})
 
-(def casebook-operation-routes
+(s/defn casebook-operation-routes [{{:keys [read-store write-store]} :StoreService
+                                    :as _services_} :- APIHandlerServices]
```
Comment (ereteog) [inline src/ctia/entity/casebook.clj:132]:
```suggestion
(s/defn casebook-operation-routes [{{:keys [read-store write-store]} :StoreService} :- APIHandlerServices]
```

**Review by ereteog — APPROVED** [review]
LGTM

File: src/ctia/entity/casebook.clj (hunk 1 above)
Comment (frenchy64) [reply src/ctia/entity/casebook.clj:132]: Keeping the `_services_` makes this process a little easier for me.

**Pull request merged**